tests/sentry/api/test_api_pagination_check.py

```diff
@@ -0,0 +1,36 @@
+from unittest import TestCase
+
+import pytest
+from django.http import HttpRequest
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint
+from sentry.api.paginator import MissingPaginationError
+
+
+class APIPaginationCheckTestCase(TestCase):
```

Comment:
we should add a test case where you name a class the same as one of the allowlists, and it should not fail